.pre-commit-config.yaml

```diff
@@ -7,4 +7,6 @@ repos:
   - repo: https://github.com/PyCQA/bandit
     rev: '1.8.3'
     hooks:
-    - id: bandit
+      - id: bandit
+        args: ['--confidence-level=high', '--severity-level=high', '-c', 'pyproject.toml']
```

Comment:
since the confidence level and and severity level are the same for CI and pre-commit can they just be set once in the pyproject.toml?

Reply:
It would be so awesome if you can figure out the syntax for that. I struggled with it for a good while and wasn't able to get to pick it up from there. Bandit's, pyproject integration is an extra and it feels a bit half-baked.

Reply:
yeah, you are right. you can read those settings from an ini file, i.e. `bandit -c pyproject.toml --ini .bandit` but event that won't work because it needs the values to be integers and will read them as strings.

Reply:
Maybe I'll make a PR over there or at least an issue